app/controllers/api/v1x0/mixins/rbac_mixin.rb

```diff
@@ -27,32 +27,11 @@ def permission_check(verb, klass = controller_name.classify.constantize)
           raise Catalog::NotAuthorized, "#{verb.titleize} access not authorized for #{klass}" unless access_obj.accessible?
         end
 
-        def permission_array_check(permissions)
```

Comment:
@syncrou 
This is all handled by openapi parser now by looking at what is allowed in openapi.json